rpm/sailfish-setup.spec

```
@@ -33,33 +33,22 @@ if ! getent passwd sailfish-actdead >/dev/null ; then
     useradd -r -g sailfish-actdead -d / -s /sbin/nologin sailfish-actdead || :
 fi
 
-# TODO : This should be moved to the first boot (JB#48049)
-# Make sure that device owner exists.
-groupadd -fg 100000 nemo || :
-if ! getent passwd nemo >/dev/null ; then
-    # Requirements - source component in brackets:
-    # video for display (setup), input input devices (systemd), audio for itself (setup)
-    # users (setup)
-    useradd -g nemo -G "video,input,audio,users" -u 100000 -m nemo || :
-fi
-
 groupadd -rf oneshot || :
-usermod -a -G oneshot nemo || :
 
 groupadd -rf sailfish-system || :
-usermod -a -G sailfish-system nemo || :
 usermod -a -G sailfish-system sailfish-mdm || :
 
 groupadd -rf ssu || :
 
 groupadd -rf sailfish-alarms || :
-usermod -a -G sailfish-alarms nemo || :
 
 groupadd -rf sailfish-datetime || :
-usermod -a -G sailfish-datetime nemo || :
 
 groupadd -rf timed || :
-usermod -a -G timed nemo || :
+
+%install
+install -D group.ids $RPM_BUILD_ROOT%{_libdir}/setup/group.ids
```

I think datadir would be better than libdir and sailfish-setup better than setup. Maybe like this:
```suggestion
install -D group.ids $RPM_BUILD_ROOT%{_datadir}/sailfish-setup/group.ids
```